### DISCUSSION TITLE
Request documentation help

### DISCUSSION THREAD

**Discussion by dnslin** [Cosium/spring-data-jpa-entity-graph#172] [Q&A] [answered]
Could you provide documentation on how to use NamedEntityGraph or DynamicEntityGraph?

Compared to building with the @NamedEntityGraph annotation on the entity class, I would like to know how to dynamically construct using NamedEntityGraph or DynamicEntityGraph to solve the N+1 problem. I look forward to your reply. Thank you.

**Comment by reda-alaoui** [comment, marked as answer]
Just read the README.md?
